pkg/imgpkg/lockconfig/images_lock.go

```diff
@@ -24,6 +24,7 @@ type ImagesLock struct {
 type ImageRef struct {
 	Image       string            `json:"image,omitempty"`       // This generated yaml, but due to lib we need to use `json`
 	Annotations map[string]string `json:"annotations,omitempty"` // This generated yaml, but due to lib we need to use `json`
+	locations   []string
```

Comment:
the relationship between Image and locations is not quite clear. i left another comment to chat about where we would be looking for assets. let's talk about this as well.